lib/src/markdown_processor.dart

```diff
@@ -300,7 +301,17 @@ Element _getRefElementFromCommentRefs(
       bool isConstrElement = ref.identifier.staticElement is ConstructorElement;
       // Constructors are now handled by library search.
       if (!isConstrElement) {
-        return ref.identifier.staticElement;
+        Element refElement = ref.identifier.staticElement;
+        if (refElement is PropertyAccessorElement) {
+          // yay we found an accessor that wraps a const, but we really
+          // want the top-level field itself
+          refElement = (refElement as PropertyAccessorElement).variable;
+        }
+        if (refElement is PrefixElement) {
+          // We found a prefix element, but what we really want is the library element.
+          refElement = (refElement as PrefixElement).enclosingElement;
```

Comment:
This is actually a second bug that was lurking behind the first one: commentRefs can contain PrefixElements.